### ISSUE TITLE
[Accessibility] Filter pills are low contrast

### ISSUE THREAD

**Issue by PeterSaathoff-Harshfield** [ampproject/amp.dev#4509]
Environment:
Any OS/browser

Steps to reproduce:
1. Open https://amp-dev-staging.appspot.com/page-experience/
2. Enter a valid URL and press Analyze

Under the "Take action. Improve your AMP site" heading, observe the grey filter pills at the bottom of each section.

Text color: #CCCCCC
Contrast ratio: 1.61:1

The color contrast ratio should be at least 4.5:1. See [WCAG 1.4.3](https://www.w3.org/TR/WCAG21/#contrast-minimum) and [Understanding Success Criterion 1.4.3](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

![image](https://user-images.githubusercontent.com/30604285/92979700-55a63500-f448-11ea-85a3-d7415c681448.png)
